cg/meta/clean/demultiplexed_flow_cells.py

```diff
@@ -60,12 +64,14 @@ def __init__(
         self._files_exist_on_disk = None
         self._passed_check = None
         self._sequencer_type = None
+        self._is_demultiplexing_ongoing_or_started = None
 
     @property
-    def sequencer_type(self) -> str:
+    def sequencer_type(self) -> Optional[str]:
```

Comment:
```suggestion
    def sequencer_type(self) -> str:
```
Will it not always return a str now?